### PR TITLE
Reorder sharing section

### PR DESCRIPTION
**Pull request by jvillafanez**
## Description
Reorder sharing section a bit

## Related Issue

- ref https://github.com/owncloud/core/issues/32705

## How Has This Been Tested?

manually tested in master

## Screenshots (if appropriate):
screen shot 2018-09-14 at 11 13 08

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [x] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] Backport (if applicable set "backport-request" label and remove when the backport was done)

### CODE REVIEW

**Comment by PVince81** [comment]
Code looks good. @pmaier1 are you ok with the screenshot ?

Note: we will miss translations if we merge this for 10.0.10

**Review by pmaier1 — APPROVED** [review]
THX!

File: apps/files_sharing/templates/settings.php [hunk 1]
```diff
@@ -22,11 +22,11 @@
...
 <div class="section" id="files_sharing">
-	<h2 class="app-name"><?php p($l->t('Files Sharing')); ?></h2>
+	<h2 class="app-name"><?php p($l->t('Group Sharing Blacklist')); ?></h2>
 	<div class="indent">
-		<p><?php p($l->t('Exclude groups from receiving shares.')); ?></p>
+		<p><?php p($l->t('Exclude groups from receiving shares')); ?></p>
 		<input name="blacklisted_receiver_groups" class="noautosave" value="<?php p($_['blacklistedReceivers']) ?>" style="width: 400px"/>
 		<br />
-		<em><?php p($l->t('These groups will not receive shares. Members of the group can still send and receive shares outside of the group.')); ?></em>
+		<em><?php p($l->t('These groups will not be available to share with. Members of the group are not restricted in initiating shares and can receive shares with other groups they are member of as usual.')); ?></em>
```
Comment (phil-davis) [inline apps/files_sharing/templates/settings.php]: s/they are member of/they are a member of/

If you think it is worth CI all running again to insert an "a"

@jvillafanez @PVince81 or @pmaier1

**Comment by jvillafanez** [comment]
Backport in https://github.com/owncloud/core/pull/32707

**Review by phil-davis — APPROVED** [review]
Message text looks good now.

**Pull request merged**